modules/messages/src/lib.rs

```diff
@@ -208,7 +208,6 @@ pub mod pallet {
 		>>::MessagesDeliveryProof;
 
 	#[pallet::pallet]
-	#[pallet::generate_store(pub(super) trait Store)]
```

Comment:
Please review this part, I'm not sure here.

Reply:
`#[pallet::generate_store(pub(super) trait Store)]` this means that if you need to generate a `Store` trait to access the storage.